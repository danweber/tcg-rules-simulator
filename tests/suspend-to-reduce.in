VERSION:2

# When one of your green Monster would evolve, by suspending 1 of your Monster, reduce the evolution cost by 5.


FIELD1:CanSusp,CS3-05 Second,CS3-02
EGGZONE1:Egg,CS3-01
HAND1:CS3-02 CS3-05

GAME:P1 T4 MAIN 9

DUMP:EGGZONE

ECHO: -- don't see breeding

CMD:EVO CS3-02 Egg
STEP:AUTOANSWER:CAPTURE
DUMP:EGGZONE

ECHO: -- evo, pay cost, suspending self
CMD:EVO CS3-05 Second
STEP:AUTOANSWER:CAPTURE
DUMP:EGGZONE
