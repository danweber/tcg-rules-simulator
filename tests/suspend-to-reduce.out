FIELD1:CanSusp,CS3-Enchantowl Second,CS3-Fairyfeatherlink
FIELD2:
EGGZONE1:Egg,CS3-Beaklet
EGGZONE2:
GAME:P1 T4 MAIN 9
ECHO: -- don't see breeding
FIELD1:CanSusp,CS3-Enchantowl Second,CS3-Fairyfeatherlink
FIELD2:
EGGZONE1:Egg,CS3-Fairyfeatherlink,CS3-Beaklet
EGGZONE2:
GAME:P1 T4 MAIN 9
ECHO: -- evo, pay cost, suspending self
CAPTURE:Activate EnchantOwl suspending
CAPTURE:Don't pay/activate
CAPTURE:Target EnchantOwl 1
CAPTURE:Target FairyFeatherlink 2
FIELD1:CanSusp,CS3-Enchantowl,REST Second,CS3-Enchantowl,CS3-Fairyfeatherlink
FIELD2:
EGGZONE1:Egg,CS3-Fairyfeatherlink,CS3-Beaklet
EGGZONE2:
GAME:P1 T4 MAIN 8
